include/secp256k1.h

```diff
@@ -370,6 +371,21 @@ SECP256K1_API int secp256k1_ec_pubkey_serialize(
     unsigned int flags
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4);
 
+/** Compare two public keys using lexicographic (of compressed serialization) order
```

Comment:
Is there more than one type of lexicographic order in this case? On a sidenote, this would not be right place to fix that because I cherry-picked this from upstream, which has already merged these functions.